docs/install/tutorials/rpi.rst

```diff
@@ -251,11 +251,11 @@ You can use :guilabel:`CTRL` + :guilabel:`SHIFT` + :guilabel:`V` to paste text i
 Installing Kolibri
 ------------------
 
-**Firstly**, follow the main instructions for installing :ref:`Kolibri on Raspberry Pi <rpi>`.
+**First**, follow the main instructions for installing :ref:`Kolibri on Raspberry Pi <rpi>`.
 
-After completing the installation, you can make kolibri available on port ``:80`` in addition to ``:8080``. This will make it possible to type ``kolibri.library`` in the browser location bar, and because of our captive portal, it will display.  
+After completing the installation, you can make kolibri available on port ``:80`` in addition to ``:8080``. This will make it possible to type, for example, a domain ``box.lan`` in the browser location bar, and because of our captive portal, it will display.  
```

Comment:
Isn't `box.lan` the default of IIAB? I understand that `library` can be difficult to spell, maybe `kolibri.lan` as a compromise?

Reply:
Couldn't it be simply `kolibri` (similar to `localhost`)?

I don't think a TLD is actually necessary, and I see some indication that [it should be avoided](https://serverfault.com/questions/17255/top-level-domain-domain-suffix-for-private-network).

Reply:
It's to avoid a browser going for it's default search engine. Typing just "kolibri" will create a Google search, rather than going to the DNS=>captive portal. I've tested it.

Reply:
true. you would have to type `http://kolibri` which is not good

Reply:
Ok, will leave `kolibri.lan` as our default, and add a tip that the domain name can be set up to a different value according to the demands of their specific implementation.